### PR TITLE
Improvements to code2parquet transform

### PR DESCRIPTION
**Pull request by daw3rd**
## Why are these changes needed?
We now only have a single python transform that can run independent of the python runtime.
Code2ParquetTransform class could not be run itself and required the python-runtime-specific sub-class to initialize some fields.  

1. Fix readmes.
2. Remove Code2PargetPythonTransform.
3. Change code2ParquetTransfrom initializer to do a better job of error checking and allow for the map of languages to be added. 
4. Define constants for keys and cli keys and some default constants
5. Don't print s3 keys
6. Continue to parse but deprecate domain and snapshot cli params, which were  never used.

## Related issue number (if any).

#238

### CODE REVIEW

**Review by sapthasurendran — COMMENTED** [review]
domain and snapshot keys are required for code datasets. Please add it back.

**Review by shivdeep-singh-ibm — APPROVED** [review]

File: transforms/code/code2parquet/python/README.md [hunk 1]
```diff
@@ -66,136 +69,60 @@ Each file contained within the ZIP is transformed into a distinct row within the
...
+  --code2parquet_supported_langs_file CODE2PARQUET_SUPPORTED_LANGS_FILE
+                        Path to file containing the list of supported languages
+  --code2parquet_detect_programming_lang CODE2PARQUET_DETECT_PROGRAMMING_LANG
+                        generate programming lang
+ ```
+
+These correspond to the configuration keys described above.
+
+### Running the samples
+To run the samples, use the following `make` targets
```
Comment (sapthasurendran) [inline transforms/code/code2parquet/python/README.md:104]: Please add make venv command before running the below commands.

Comment (daw3rd) [reply transforms/code/code2parquet/python/README.md:104]: This is a good point, although when running `make run-cli-sample`, for example, it prints an error message indicating `make venv` needs to be run.

**Comment by daw3rd** [comment]
> domain and snapshot keys are required for code datasets. Please add it back.

But these were never used in transform().  THey were parsed and I left that in, just to keep kfp happy.

**Pull request merged**